dtrace-provider.js

```diff
@@ -10,33 +10,32 @@ DTraceProviderStub.prototype.enable = function() {};
 DTraceProviderStub.prototype.fire = function() {};
 DTraceProviderStub.prototype.disable = function() {};
 
+var reqmode = process.env.NODE_DTRACE_PROVIDER_REQUIRE;
 var builds = ['Release', 'default', 'Debug'];
 var err = null;
 
-for (var i in builds) {
-    try {
-        var binding = require('./build/' + builds[i] + '/DTraceProviderBindings');
-        DTraceProvider = binding.DTraceProvider;
-        break;
-    } catch (e) {
-        if (err === null) {
-            err = e;
+if (reqmode === 'skip') {
```

Comment:
What would be a use case where a user would want the `'skip'` behavior instead of the default one?

Reply:
If you were switching between 32- and 64-bit node, then the require would cause the process to abort. It's possible that the skip mode isn't really necessary though, and in that case you should just rebuild the modules. I can remove this if it seems unnecessary.

Reply:
Doesn't the default mode wraps the `require` call with a `try/catch` block below and discard the error though?

Reply:
Yes. I think that `skip` is probably unnecessary, and the `try/catch` block will suffice.